### PR TITLE
test(dropdown): updated tests

### PR DESCRIPTION
**Pull request by mario-subo**
removed enzyme in favour of react test utils

TODO:
- [x] Replace existing tests with react test utils
- [x] Add missing tests

### CODE REVIEW

**Comment by mario-subo** [comment]
image

**Comment by mario-subo** [comment]
image

File: src/Dropdown/Dropdown.test.tsx [hunk 1]
```diff
@@ -1,213 +1,820 @@
...
+            clearable: true,
+            disabled: true,
+            selectedValue: { ...DEFAULT_PROPS.list[0] }
+        };
+
+        act(() => {
+            render(<Dropdown {...props} />, container);
+        });
+
+        const clearButton: Element = container.querySelector(".right-items > .dropdown-icon-holder:not(.chevron");
```
Comment (DanSallau) [inline src/Dropdown/Dropdown.test.tsx]: How does this work when you seem to be missing a close bracket ?

File: src/Dropdown/Dropdown.test.tsx [hunk 2]
```diff
@@ -1,213 +1,820 @@
...
+        };
+
+        act(() => {
+            render(<Dropdown {...props} />, container);
+        });
+
+        const target: Element = container.querySelector("#select-all");
+        expect(target).toBeTruthy();
+
+        const selectedOptions: NodeListOf<Element> = container.querySelectorAll(".custom-dropdown-item.selected");
```
Comment (DanSallau) [inline src/Dropdown/Dropdown.test.tsx]: You aren't reusing anywhere , why the variable ?

File: src/Dropdown/Dropdown.test.tsx (hunk 1 above)
Comment (mario-subo) [reply src/Dropdown/Dropdown.test.tsx]: ohh the closing bracket inside the string was missing, thanks.

File: src/Dropdown/Dropdown.test.tsx (hunk 2 above)
Comment (mario-subo) [reply src/Dropdown/Dropdown.test.tsx]: you're right

**Pull request merged**